CotEditor/tr.lproj/WindowPane.strings

```diff
@@ -29,11 +29,11 @@
 /* Class = "NSMenuItem"; title = "Respect System Setting"; ObjectID = "Guj-Kc-zf2"; */
 "Guj-Kc-zf2.title" = "Sistem Ayarını Koru";
 /* Class = "NSMenuItem"; title = "Never"; ObjectID = "o1f-vR-aFm"; */
-"o1f-vR-aFm.title" = "her zaman";  // The same as the popup menu selection in the System Preferences > "General"
+"o1f-vR-aFm.title" = "asla";  // The same as the popup menu selection in the System Preferences > "General"
 /* Class = "NSMenuItem"; title = "Automatically"; ObjectID = "10Z-Xd-CLV"; */
-"10Z-Xd-CLV.title" = "Kendiliğinden";
+"10Z-Xd-CLV.title" = "otomatik";
 /* Class = "NSMenuItem"; title = "Always"; ObjectID = "OJs-2q-bg5"; */
-"OJs-2q-bg5.title" = "Her zaman";  // The same as the popup menu selection in the System Preferences > "General"
+"OJs-2q-bg5.title" = "her zaman";  // The same as the popup menu selection in the System Preferences > "General"
 
```

Comment:
Thank you for the fix. Unlike in the System Preferences, we use the title case capitalization for these menu items (I feel the System Preferences made a mistake. In the standard rule, the first word should be capitalized when the line follows a semicolon). Could you update if there is no extra reason?

Reply:
Fixed.